src/menu/tray.js

```diff
@@ -83,17 +84,65 @@ export class Tray extends EventEmitter {
     if (!kubeconfigPath) {
       throw new Error('No kubeconfig path found');
     }
-    fs.watch(kubeconfigPath, () => {
-      this.updateContexts();
-      const contextMenu = electron.Menu.buildFromTemplate(this.#contextMenuItems);
-
-      this.#trayMenu.setContextMenu(contextMenu);
+    // Add a delay before starting the file system watcher to avoid hitting a lock on the file
+    // that is taking a while to be cleared.
+    // setTimeout(() => {
+    // Maybe the file exists now...
```

Comment:
Shouldn't this be removed since it's not in use?

Reply:
removed